examples/connectivity/plot_adhd_covariance.py

```diff
@@ -8,11 +8,26 @@
  
 plotted_subject = 0  # subject to plot
 
+import numpy as np
 
 import matplotlib.pyplot as plt
+
+from nilearn import plotting, image
 from nilearn.plotting import cm
 
 
+def plot_connectome(cov, atlas_maps, **kwargs):
+    """Plot connectome given a covariance matrix and atlas maps"""
+    imgs = image.iter_img(atlas_maps)
+    regions_coords = np.array([
+        map(np.asscalar, plotting.find_xyz_cut_coords(img)) for img in imgs])
+    np.random.seed(42)
+    node_colors = np.random.rand(len(atlas_maps), 3)
+    plotting.plot_connectome(cov, regions_coords,
+                             nodes_kwargs={'s': 50, 'c': node_colors},
```

Comment:
I would prefer these get changed to `size` and `color`, or to follow @GaelVaroquaux suggestion of eliminating the `nodes_kwargs` and making them meaningful parameters in the function interface.